R/ggsurvfit.R

```diff
@@ -211,16 +211,15 @@ ggsurvfit <- function(x, type = "survival",
 
   # return time label ----------------------------------------------------------
   switch( # using the CDISC variable as default label, if present
-    isTRUE(any(c("PARAM", "PARAMCD") %in% names(data))),
+    isTRUE(!is.null(data) && any(c("PARAM", "PARAMCD") %in% names(data))),
```

Comment:
You would need to check two things:
1. is the param/paramcd unique in the data used for estimation. If not, then the programmer forgot to make a subset
2. if param or paramcd was used as strata then label should default to Time